lib/common/formatHelpers/createPropertyFormatter.js

```diff
@@ -19,6 +19,59 @@ const defaultFormatting = {
   suffix: ';'
 }
 
+/**
+ * Split a string comment by newlines and
+ * convert to multi-line comment if necessary
+ * @param {string} to_ret_prop
+ * @param {{comment: string; style: 'short' | 'long'; indentation: string}} options
+ * @param {'short' | 'long'} commentStyle
+ * @param {string} indentation
+ * @returns {{hasNewLines: boolean, comment: string}}
+ */
+function addComment(to_ret_prop, options) {
+  const { comment, style, indentation } = options;
+
+  const [commentStyle,placeAbove] = style.split('-above');
```

Comment:
Should we just have a separate option for `commentPosition`? Feels a bit weird to split a string for configuration

Reply:
Agree, was a bit hacky, I improved the code to use commentPosition, also improved the switch statement